### PR TITLE
Feature - TSS-1004 - Change regional priority to overseas priority

### PR DESCRIPTION
**Pull request by chris-pettinga**
changing regional priority reference to overseas priority

### CODE REVIEW

**Review by osimuka — APPROVED** [review]

**Comment by Santino-Trade** [comment]
Sprint 42 Relese 2 merge order (to go Monday 2nd October):
1. feature/tss-1004/change-regional-to-overseas into TSS-993_Priority_Page_PB100-Regional_Rollup
https://github.com/uktrade/market-access-python-frontend/pull/518
2. TSS-993_Priority_Page_PB100-Regional_Rollup into rc/sprint-42-a-new-hope-2
https://github.com/uktrade/market-access-python-frontend/pull/515
3. feature/tss-966/create-scoping-tag into rc/sprint-42-a-new-hope-2
https://github.com/uktrade/market-access-python-frontend/pull/516
4. rc/sprint-42-a-new-hope-2 into master

**Review by Santino-Trade — APPROVED** [review]

File: barriers/views/edit.py [hunk 1]
```diff
@@ -191,10 +191,18 @@ def get_initial(self):
             self.barrier.top_priority_status or TOP_PRIORITY_BARRIER_STATUS.NONE
         )
 
+        if self.barrier.top_priority_status in [
+            TOP_PRIORITY_BARRIER_STATUS.APPROVAL_PENDING,
+            TOP_PRIORITY_BARRIER_STATUS.REMOVAL_PENDING,
+            TOP_PRIORITY_BARRIER_STATUS.APPROVED,
+        ]:
+            existing_priority_level = "PB100"
```
Comment (chris-pettinga) [inline barriers/views/edit.py:199]: But we have not added PB100 as a choice on the form?

File: barriers/forms/edit.py [hunk 2]
```diff
@@ -319,18 +319,27 @@ def save(self):
...
     )
-    country_help_text = "Actively being worked on by you or your team."
+    country_help_text = "Category currently under review. Please do not assign new barriers as Country priority."
     watchlist_help_text = "Of potential interest but not actively being worked on."
     CHOICES = [
         (
-            "REGIONAL",
-            "<span class='govuk-body'>Regional priority</span> <span"
-            f" class='govuk-hint'>{regional_help_text}</span>",
+            "PB100",
```
Comment (Santino-Trade) [inline barriers/forms/edit.py:335]: Its added as an option here

File: barriers/views/edit.py (hunk 1 above)
Comment (Santino-Trade) [reply barriers/views/edit.py:199]: https://github.com/uktrade/market-access-python-frontend/pull/518/files#r1340124363

**Pull request merged**